src/workerd/server/tests/python/sdk/worker.py

```diff
@@ -1,8 +1,12 @@
-from http import HTTPMethod
+from http import HTTPMethod, HTTPStatus
 
-from cloudflare.workers import Response, fetch
+import js
 
+from cloudflare.workers import FormData, Response, fetch
 
+
+# Each path in this handler is its own test. The URLs that are being fetched
+# here are defined in server.py.
 async def on_fetch(request):
     if request.url.endswith("/modify"):
```

Comment:
Would be nice to refactor this to use a dispatch table and a decorator.

Reply:
Maybe I'll make a followup PR to rearrange this set of tests into a tidier form.